sdk/java-sdk/src/main/scala/kalix/javasdk/impl/workflowentity/WorkflowEntityImpl.scala

```diff
@@ -216,7 +220,7 @@ final class WorkflowEntityImpl(system: ActorSystem, val services: Map[String, Wo
           Future.failed(ProtocolException(command, "Receiving Workflow is not the intended recipient of command"))
 
         case InCommand(command) if command.payload.isEmpty =>
-          Future.failed(throw ProtocolException(command, "No command payload for Workflow"))
```

Comment:
Wow, that's weird. 

Can you check the other impl? I think we copy it form ValueEntity.

Reply:
other entities are fine